src/eval-frame/components/reps/error-handler.jsx

```diff
@@ -2,129 +2,6 @@ import React from "react";
 import styled from "react-emotion";
 import PropTypes from "prop-types";
 
-import ErrorStackParser from "error-stack-parser";
```

Comment:
broke the error stack parser into a separate file from the component so that it can be used remotely